README.rst

```diff
@@ -609,6 +609,39 @@ Cache can be disabled via ``--using-cache`` option or config file:
         ->setUsingCache(false)
     ;
 
+Cache file can be specified via ``--cache-file`` option or config file:
+
+.. code-block:: php
+
+    <?php
+
+    return Symfony\CS\Config\Config::create()
+        ->setCacheFile(__DIR__.'/.php_cs.cache')
+    ;
+
+Speed up builds on Travis
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Setting the cache file can be very useful for speeding up your builds on Travis (when
+running in private repositories and on container-based infrastructure, see
+http://docs.travis-ci.com/user/caching/), as you could specify a cache file in a
+dedicated directory and have Travis cache that directory between builds.
+
+.. code-block:: bash
+
+    language: php
+    php:
+      - 5.5
+    sudo: false
+    cache:
+      directories:
+        - "$HOME/.php-cs-fixer"
+    before_script:
+      - mkdir -p "$HOME/.php-cs-fixer"
+    script:
+      - vendor/bin/php-cs-fixer fix --cache-file "$HOME/.php-cs-fixer/.php_cs.cache" --dry-run --diff --verbose
```

Comment:
if one send a PR for `foo` branch, is is possible to store cache inside `$HOME/.php-cs-fixer/.php_cs.cache.foo` file ?